conf/distro/lmp-base.conf

```diff
@@ -0,0 +1,18 @@
+require conf/distro/include/lmp.inc
+
+DISTRO = "lmp-base"
+DISTRO_NAME = "Linux-microPlatform Base (no ostree)"
+
+IMAGE_LINGUAS ?= "en-us"
+
+# No graphical feature as part of the base platform
+DISTRO_FEATURES_remove = "wayland x11"
+
+# By default we only support initramfs. We don't build live as that
+# pulls in a lot of dependencies for the live image and the installer, like
+# udev, grub, etc.  These pull in gettext, which fails to build with wide
+# character support.
+IMAGE_FSTYPES = "cpio.gz"
+QB_DEFAULT_FSTYPE = "${IMAGE_FSTYPES}"
+
+MACHINE_ESSENTIAL_EXTRA_RDEPENDS = ""
```

Comment:
I should probably comment here something like:
By default we don't have any extra machine dependencies